### ISSUE TITLE
[Bug] Can't create a section with level bar

### ISSUE THREAD

**Issue by bedelaitre** [AmruthPillai/Reactive-Resume#1741]
### Is there an existing issue for this?

- [X] Yes, I have searched the existing issues and none of them match my problem.

### Product Variant

Self-Hosted

### Current Behavior

4.0.4 version - I am not able to create a section like Skills, Languages with colored level bars

### Expected Behavior

Be able to create a section like Skills, Languages with colored level bars

### Steps To Reproduce

1. Goto a Resume
2. Create a new section
==> There is no option to have colored level bars

### What browsers are you seeing the problem on?

Microsoft Edge

### What template are you using?

None

**Comment by s1dd-tech** [comment]
Yes, this ability was there in the previous build, but is missing now. Also, is there a possibility to add a varied scale options instead of 5 to 8. Or make it a decimal level entry.

**Comment by amruthpillai** [comment]
Closing this issue as I'm doing some clean up to move to a newer version of the project. I apologize in advance for the inconvenience if you find the need to reopen this issue.

Please read #2499 for more information about the migration.
